modules/ROOT/partials/configuration/markdown.adoc

```diff
@@ -0,0 +1,34 @@
+[[markdown_symbols]]
+== `markdown_symbols` option
+
+The `markdown_symbols` option allows you to define `key/value` pairs where the `key` represents a symbol in your content and the `value` represents the replacement for that symbol.
+
+*Type:* `+Object+`
+
+*Default value:* The `+markdown_symbols+` object excepts the following key-value pairs by default currently:
+
+[source, js]
+----
+{
+  C: '©',
+  TM: '™',
+  R: '®'
+}
+----
+
+=== Example: using `markdown_symbols`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  markdown_symbols: {
```

Comment:
@HAFRMO 

1. Do you expect "markdown_symbols:" config to work? if so please attach a fiddle with sample data. 
2. Any other config options related to markdown plugin?

Reply:
We need to mention that the key symbols, like `C`, should be embraced with parenthesis, like `(C)` in the Markdown content and they are case-sensitive as well, `(c)` will not be converted.

Reply:
@vpyshnenko - Added a note in the latest commit to cover this.